hash_practice/exercises.py

```diff
@@ -2,18 +2,30 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: 
+        O(m * n log n) with m number of words and n average number of chars in a word
+        
+        Space Complexity: O(m * n)
     """

```

Comment:
👍 The time complexity is correct, although if you assume the words are limited in length you can say it's O(n), but that's an assumption.